pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.ruff]
-select = [
+lint.select = [
```

Comment:
can you rather move these to a new `[tool.ruff.lint]` section?

Reply:
Done.